pennylane/ops/qubit/parametric_ops.py

```diff
@@ -848,15 +964,42 @@ def _matrix(cls, *params):
         )
 
     @staticmethod
-    def decomposition(theta, wires):
-        decomp_ops = [
+    def compute_decomposition(phi, wires):
+        """Determine ``CRX``'s decomposition for specified wire.
+        The decomposition defines an Operator as a product of more fundamental gates:
+
+        .. math:: O = O_1 O_2 \dots O_n.
+
+        ``compute_decomposition`` is a static method and can provide the decomposition of a given
+        operator without creating a specific instance. The instance method ``decomposition`` uses
+        this method and the instance's variables.
+
+        Args:
+            phi (float): rotation angle :math:`\phi`
```

Comment:
I was wondering, do we need r""" if we use inline math?

Reply:
By the way, I haven't checked all rendered docstrings in other PRs, we can do this before merging the base branch